dotcom-rendering/src/model/enhanceTags.ts

```diff
@@ -0,0 +1,24 @@
+import type { FETagType, TagType } from '../types/tag';
+
+export const enhanceTags = (tags: FETagType[]): TagType[] =>
+	tags.map((tag) => enhanceTag(tag));
+
+const enhanceTag = ({ properties }: FETagType): TagType => {
+	const {
+		id,
+		tagType,
+		webTitle,
+		twitterHandle,
+		bylineImageUrl,
+		contributorLargeImagePath,
+	} = properties;
+
+	return {
+		id,
+		type: tagType,
+		title: webTitle,
+		twitterHandle,
+		bylineImageUrl,
+		bylineLargeImageUrl: contributorLargeImagePath,
+	};
+};
```

Comment:
Not a blocker, but wondering if we could destructure and do the key renaming in one go? I think it has a slight better balance of signal to noise ratio, and now that you have added some tests we can be more confident in refactoring 🫶 ! 

```ts
const enhanceTag = ({
    properties: {
        id,
        tagType: type,
        webTitle: title,
        twitterHandle,
        bylineImageUrl,
        contributorLargeImagePath: bylineLargeImageUrl,
    },
}: FETagType): TagType => ({
    id,
    type,
    title,
    twitterHandle,
    bylineImageUrl,
    bylineLargeImageUrl,
});
```

Reply:
Ah yes that's nice, thanks!